tools/README.md

```diff
@@ -32,6 +32,12 @@ This tool is used to distribute the following items:
 - Destrier Beret  
 - Chocobo Shirt  
 
+## Announce
+`python accounce.py "<your message>"`
```

Comment:
Thank you, coco. I think a small typo made his way there.

Reply:
Whoops, thank you!